packages/flutter/lib/src/material/refresh_indicator.dart

```diff
@@ -122,6 +122,7 @@ class RefreshIndicator extends StatefulWidget {
     this.semanticsValue,
     this.strokeWidth = 2.0,
     this.triggerMode = RefreshIndicatorTriggerMode.onEdge,
+    this.isRefreshing
```

Comment:
You could add a default parameter here as false and remove the optional.

Reply:
This parameter is nullable to avoid breaking change with the method `Future<void> show({ bool atTop = true })`.

If the parameter have value false, it can hide the progress indicator when the method show above need to make the progress indicator run. It does not make sense to use both methods to do the same thing. So its nullable only to avoid 
a breaking change.